vtex-apps/b2b-organizations
# Feat/add warning holding organization

#### What problem is this solving?
The idea is to allow merchants to search for the ID and Trade Name fields on the screen that lists organizations in the admin (admin/b2b-organizations/organizations#/organizations). This implementation has already been done, however, if we publish without preparation, we will prevent accounts from working until they adjust some settings. This activity aims to prepare the accounts for this change.
  
 #### How to test it?
To do this, we must instruct the merchant to check whether it is possible to search by organization ID and trade name (we recommend reading this page to understand more about the subject). By default, the ID field is usually already indexed. However, the trade name field does not. You for changing this setting via Advanced Settings in Masterdata
Data Structure -> Data Entities -> Edit Customer (CL) -> Procurar por Trade Name e clicar em editar.
  
  
  
  [Workspace](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-organizations/organizations#/organizations)
  
  #### Screenshots or example usage:
<img width="1440" alt="Screenshot 2024-10-14 at 16 12 56" src="https://github.com/user-attachments/assets/369553cc-e349-4cab-85e4-845072d7081d">



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
